core/src/test/scala/com/rackspace/com/papi/components/checker/step/ResultSuite.scala

```diff
@@ -27,7 +27,7 @@ class ResultSuite extends BaseStepSuite {
   val e12 = new MismatchResult("Mismatch", StepContext(-1), "12")
   val e13 = new MismatchResult("Mismatch", StepContext(-1), "13")
   val e14 = new BadMediaTypeResult("Bad Media Type", StepContext(-1), "14", 8)
-  val e15 = new MethodFailResult("Bad Method", StepContext(-1), "15", 8, Map("Foo"->"Bar"))
+  val e15 = new MethodFailResult("Bad Method", StepContext(-1), "15", 7, Map("Foo"->"Bar"))
```

Comment:
I changed this priority to fix this test. When multiple `ErrorResult`s are reported, they are aggregated in a `PriorityQueue` within a `MultiFailResult`. If they have the same priority, one will be selected at random (since ordering within a "bucket" of a priority queue is not guaranteed). So instead, we can just assign priorities to remove the random aspect of this test.

Oh, and to be clear, the ordering changed with the upgrade to Scala 2.12 which was causing tests to fail.